app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsContributionDetailsFragment.kt

```diff
@@ -60,7 +60,7 @@ class SuggestedEditsContributionDetailsFragment : Fragment() {
         updateTopGradient()
         contributionContainer.setOnClickListener { startTypeSpecificActivity() }
         contributionDetailText.text = contribution.description
-        revisionText.text = if (contribution.top) getString(R.string.suggested_edits_contribution_current_revision) else contribution.revisionId.toString()
+        if (contribution.top) revisionText.text = getString(R.string.suggested_edits_contribution_current_revision) else revisionLayout.visibility = GONE
```

Comment:
Should we set the text in XML and use this logic to make the view `VISIBLE` or `GONE`?

Reply:
correct! done.